packages/react/tsconfig.json

```diff
@@ -8,6 +8,6 @@
     "types": ["vitest/globals", "vite/client"]
   },
   "include": ["src", "cunningham.ts"],
-  "exclude": ["node_modules", "dist", "**/tokens.ts", "**/*.stories.tsx"],
+  "exclude": ["node_modules", "dist", "**/tokens.ts"],
```

Comment:
hum, is finally see that you removed this configuration in purpose.
Has you prefer.